cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -69,14 +69,14 @@ object AnalyzerCommand : CommandWithHelp() {
     @Suppress("LateinitUsage")
     private lateinit var inputDir: File
 
-    @Parameter(description = "The directory to write the analyzer result file to.",
+    @Parameter(description = "The directory to write the analyzer result as ORT result file(s) to.",
```

Comment:
Are there cases where an analyzer result is written in a different way, e.g. not as ORT result?  The new description suggests to me that there are multiple ways of writing the analyzer result.

If all results always go into an 'ORT result' would it make sense to not differentiate at all and just say 'The directory to write the result file to'?

Reply:
No, there are no other ways to write analyzer results than as part of a ORT result file. I wanted to emphasize the opposite direction: There is no dedicated analyzer result file, it's always an ORT result file which contains an analyzer result.

Reply:
Maybe make it more explicit:

> The directory to write the analyzer result as ORT result file to. If multiple output formats are defined multiple files are written.